### DISCUSSION TITLE
Table layout Split: missing headers on desktop

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#5106] [Help] [answered]
Hello,

I notice that when using a Split table layout, the headers doesn't show also on desktop. This is a problem when a table has two similar columns (like created at and updated at, or a creator and a responsible user names) because it's not possible to know which column is one or another.

Is there a way to enable the headers on desktop?

Also, it would be good for these situations, on mobile, to have the column title printed above every field on a smaller and gray font, or maybe a first column with the titles on every row.

**Comment by danharrin** [comment, marked as answer]
When using table layouts, your table is not a table any more. The concept of table column headers would be inside the `<thead>`, but this doesnt exist outside of real tables.

You could maybe try `->description('Created at', position: 'above')` on the column to add a label?

**Comment by underdpt** [comment]
That's definitely a good solution! Thanks!

**Reply by photojo** [reply]
Sorry, when my question is already answered, but is there a way to use panel with collapse and keep all headers in table?
